tap/source/packet_source_manager.go

```diff
@@ -0,0 +1,140 @@
+package source
+
+import (
+	"fmt"
+	"runtime"
+	"strconv"
+	"strings"
+
+	"github.com/up9inc/mizu/shared/logger"
+	"github.com/vishvananda/netns"
+)
+
+type PacketSourceManager struct {
+	sources []*tcpPacketSource
+}
+
+func NewPacketSourceManager(procfs string, pids string, filename string, interfaceName string,
+	behaviour TcpPacketSourceBehaviour) (*PacketSourceManager, error) {
+	sources := make([]*tcpPacketSource, 0)
+	host_source, err := newHostPacketSource(filename, interfaceName, behaviour)
+
+	if err != nil {
+		return nil, err
+	}
+
+	sources = append(sources, host_source)
```

Comment:
```suggestion
	sources = append(sources, hostSource)
```

Reply:
Done in https://github.com/up9inc/mizu/pull/418/commits/561f4a8b6459de69175fb9f27490b650579999f5